web/src/lib/stores/user.ts

```diff
@@ -0,0 +1,13 @@
+import { localWritable } from '$lib/store';
+import { derived } from 'svelte/store';
+
+export interface UserStore {
```

Comment:
including user ID might be helpful down the line... though none of the current endpoints require it